config.go

```diff
@@ -601,6 +717,25 @@ func (inc *Include) Get(alias, key string) string {
 	return ""
 }
 
+// GetAll finds all values in the Include statement matching the alias and the
+// given key.
+func (inc *Include) GetAll(alias, key string) ([]string, error) {
+	inc.mu.Lock()
+	defer inc.mu.Unlock()
+	// TODO: we search files in any order which is not correct
+	for i := range inc.matches {
+		cfg := inc.files[inc.matches[i]]
+		if cfg == nil {
+			panic("nil cfg")
+		}
+		val, err := cfg.GetAll(alias, key)
```

Comment:
Is this correct? Should we keep iterating and find all possible values, not just the first match?

Can we change the check below from a nil check to `if len() > 0`? It's possible for a slice to be non-nil and empty.

Reply:
The code was mostly copy/pasted from Get(). Should the nil check be removed there too?